### ISSUE TITLE
Types of Constitutions

### ISSUE THREAD

**Issue by ghost** [jackuser/write-a-constitution#3]
We need to add coverage of the following types of constitutions:
- [ ] Federalist
- [ ] Monarchy
- [x] Codified
- [x] Uncodified
- [x] Unwritten
- [x] Written
- [ ] Republic
- [ ] Democratic

**Comment by jackuser** [comment]
@BenjaminProgram  I agree that we should categorize them. It would be helpful if we could find a good working list.

**Comment by ghost** [comment]
@jackuser Let's work on these changes [here](https://github.com/jackuser/write-a-constitution/tree/part-1-working).

**Comment by jackuser** [comment]
@BenjaminProgram I will join you there.

**Comment by ghost** [comment]
I propose we merge current work in #4.
